src/instructlab/cli/rag/convert.py

```diff
@@ -59,6 +60,10 @@ def convert(
     input_dir,
     output_dir,
 ):
+    if not FeatureGating.feature_available(GatedFeatures.RAG):
```

Comment:
what about a `FeatureGating.check_feature_available(GatedFeatures.RAG)` to raise a consistent error from any client, instead of having each consumer to repeat the same check? (this would also let you keep the `env_var_name` internal to the module)

Reply:
I chose this way so that different error messages could be output in different contexts, allow for different degrees of logging, etc.

Not a strong opinion either way.

Reply:
The risk there is that those error messages can become outdated - which can be addressed with unit testing, so will do.

Reply:
Done.